src/app/layout.tsx

```diff
@@ -1,12 +1,34 @@
 import type { Metadata } from 'next'
 import { Inter } from 'next/font/google'
+import localFont from "next/font/local"
 import './globals.css'
 
+const futura = localFont(
+  {
+    src: [
+      // {
+      //   path: "../../public/fonts/futura-pt.ttf",
+      //   weight: "400",
+      // },
+      {
+        path: "../../public/fonts/futura-pt-bold.ttf",
+        weight: "700",
+      },
+      // {
+      //   path: "../../public/fonts/futura-pt-light.ttf",
+      //   weight: "400",
+      // }
+    ],
+    variable: "--font-futura",
+  }
+)
+
 // Load fontawesome css at the root of the app to stop 'flashing' icon
-import '@fortawesome/fontawesome-svg-core/styles.css';
-import { config } from "@fortawesome/fontawesome-svg-core";
+import '@fortawesome/fontawesome-svg-core/styles.css'
+import { config } from "@fortawesome/fontawesome-svg-core"
 config.autoAddCss = false;
 
+import NavBar from '@components/NavBar'
 import TopBar from '@components/TopBar'
 
 const inter = Inter({ subsets: ['latin'] })
```

Comment:
will we still use this inter font somewhere? if we're all-in on futura-pt we may as well remove it

Reply:
Well I do want to use two fonts. Futura really just for Headings etc and then a normal font for most regular text, need to play around with it